Fix:

engine/stringstream.cpp:202:12: error: case value evaluates to -1, which cannot be narrowed to type 'uint64' (aka 'unsigned long long') [-Wc++11-narrowing]

See:

https://github.com/falconpl/falcon/issues/15

Part of:

https://github.com/falconpl/falcon/commit/b37f12f4b57af193f81dc26820ed6148a8021d54
--- engine/stringstream.cpp.orig
+++ engine/stringstream.cpp
@@ -201,7 +201,7 @@ bool StringStream::errorDescription( String &description ) const
    switch( m_lastError )
    {
       case 0:  description = "None"; return true;
-      case -1: description = "Out of Memory"; return true;
+      case (long long unsigned int) -1: description = "Out of Memory"; return true;
    }
 
    return false;
